### ISSUE TITLE
Menu not working on IOS, fine on Android

### ISSUE THREAD

**Issue by ElixirMike** [instea/react-native-popup-menu#160]
I have the menu working fine in my Android environment...the top level component is wrapping with <MenuProvider>

I am now trying it in IOS, and when I click, nothing happens...the menu doesn't pop-up. In addition, it seems to freeze all the other actions in the component, other actions become unresponsive and I have to restart the application to get it back to working order. 

Any ideas or changes I need to make in IOS to get this working?? 
This is running on a Iphone simulator with version 12.2 of the IOS software.

**Comment by sodik82** [comment]
can you provide us sample project eg. as expo snack? You can use https://snack.expo.io/@sodik82/react-native-popup-menu as a base if you want.

**Comment by ElixirMike** [comment]
Here is the snack:  https://snack.expo.io/rJboKv3C4

I added the two main files.
1. sidedrawer.js  :  the main component which has the <MenuProvider>
wrapper around the sub components and the

2. UserPanel.js file is where I render the menu.

Please note this works fine in my Android environment but does not work in
my IOS environment. I tried to play around/simplify the menu structure as
much as possible...but it's just not working. When I click on it, it does
not pop-up, no error is shown in console....so not sure how to proceed.

Let me know if this is enough to reproduce/troubleshoot.

Thanks,
Mike

**Comment by sodik82** [comment]
the point of snacks is that you can actually run examples on your devices - which I can't in your case. But at least I see some code.

It looks that sidedrawer is not your root component, doesn't it? Because `MenuProvider` should be always at the very root of your application.

**Comment by ElixirMike** [comment]
It wouldn't be easy to replicate a working example here..I'm using react-native-navigation which changes how you wrap the application. Upon logging in, the Navigation is used to direct to 2 separate components...the "main" component and a sidedawer component, which is where I have <MenuProvider> wrapping.

Again, this works on android...so it appears I have the code setup properly? Could it be something with my IOS environment? Anything I can check?

**Comment by sodik82** [comment]
not necessarily. the IOS and Android handles differently z-Index and it can be the issue. hard to tell.

**Comment by jhondavila** [comment]
some solution, I present the same problem in ios platform

**Comment by sodik82** [comment]
is it problem only inside RNN side drawer or is it problem also on other screens?

**Comment by jhondavila** [comment]
it seems that only happens with RNN

**Comment by Bayramito** [comment]
+1 i got the same error today, still not working on ios...

**Comment by shiroze** [comment]
This plugin still not working on ios, someone find the solution already ?

**Comment by muhammadali543** [comment]
You can solve this issue by providing <MenuProvider> to each screen where <Menu> is using. It's not working on IOS when place in _layout file.
